### PR TITLE
export types

### PR DESCRIPTION
**Pull request by zeroXbrock**
now you can
```typescript
import type {
  SuaveWallet,
  SuaveProvider,
  TransactionRequestSuave,
  // ...
} from 'viem/chains/utils'
```

or 
```typescript
import type {
  SuaveProvider,
  SuaveWallet,
  TransactionRequestSuave
} from 'viem/_types/chains/utils'
```

### CODE REVIEW

**Review by dmarzzz — APPROVED** [review]
LSTM

File: examples/suave/bids/index.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ export class MevShareBid {
     return {
       to: this.mevShareContract,
       data: this.newBidCalldata(),
-      type: SuaveTxTypes.ConfidentialRequest,
+      type: '0x43',
```
Comment (metachris) [inline examples/suave/bids/index.ts:68]: why replace with hard-coded value instead of const?

Comment (zeroXbrock) [reply examples/suave/bids/index.ts:68]: you can use either one -- this is just more convenient, and a demonstration of what most people will likely do

**Review by andytudhope — APPROVED** [review]
thank you :+1:

**Pull request merged**